torch/_refs/fft.py

```diff
@@ -399,9 +416,9 @@ def ihfftn(
 
     if len(dim) == 1:
         tmp = _apply_norm(tmp, norm=norm, signal_numel=shape[0], forward=False)
-        return prims.conj(tmp)
+        return torch.conj_physical(tmp) if out_defined() else tmp.conj()
 
-    tmp = prims.conj_physical(tmp)
+    tmp = torch.conj_physical(tmp)
```

Comment:
Since I had to make a change here anyway, I changed `prims` to `torch` calls because those are usually preferred.